pineappl_py/src/grid.rs

```diff
@@ -383,15 +383,16 @@ impl PyGrid {
     ///     numpy.ndarray(float) :
     ///         cross sections for all bins, for each scale-variation tuple (first all bins, then
     ///         the scale variation)
+    #[pyo3(signature = (pdg_id, xfx, alphas, order_mask = None, bin_indices = None, lumi_mask= None, xi = None))]
```

Comment:
Add one space between `lumi_mask` and `=`.

Reply:
I guess running `cargo fmt` might be enough? can we convince you to activate pre-commit :innocent: ? Look e.g. at [eko](https://github.com/NNPDF/eko/blob/b33509a03cd3ff5a03e6cb535da49617d84937bf/.pre-commit-config.yaml#L46)

Reply:
> I guess running `cargo fmt` might be enough? can we convince you to activate pre-commit 😇 ? Look e.g. at [eko](https://github.com/NNPDF/eko/blob/b33509a03cd3ff5a03e6cb535da49617d84937bf/.pre-commit-config.yaml#L46)

Would it? I have automatic formatting (which should be running `cargo fmt` under the hood) when saving on my helix editor but it wasn't able to pick that up.

Reply:
It wouldn't, because `cargo fmt` won't change the contents of macros it doesn't know, because there whitespace may make a difference.